tox.ini

```diff
@@ -0,0 +1,23 @@
+[tox]
+envlist=
+    py27,
+    py34,
+    py35
+
+[flake8]
+max-line-length= 100
+exclude= tests/*
+
+[testenv]
+commands=py.test --tb native {posargs:tests}
```

Comment:
It recursively searches the project for something like `**/tests/test_*.py`.  That is probably not the exact glob pattern but you get the idea.